sdk/push/src/test/java/com/klaviyo/push/KlaviyoPushServiceTest.kt

```diff
@@ -4,10 +4,12 @@ import android.content.Context
 import android.content.SharedPreferences
 import com.klaviyo.coresdk.Klaviyo
 import com.klaviyo.coresdk.KlaviyoConfig
-import com.klaviyo.coresdk.networking.KlaviyoCustomerProperties
 import com.klaviyo.coresdk.utils.KlaviyoPreferenceUtils
 import com.klaviyo.push.KlaviyoPushService.Companion.PUSH_TOKEN_PREFERENCE_KEY
-import io.mockk.*
```

Comment:
pre-commit failed me on having a .* import even though I was just merging master into my branch.. idk man.

Reply:
Wait we're running pre-commit checks on the Android SDK?

Reply:
I think kenny added it recently, it runs KTLint

Reply:
Oh ok, cool!!

Reply:
https://klaviyo.atlassian.net/wiki/spaces/EN/pages/3601793078/Android+Development+Setup#Steps in there

Reply:
yup sorry i think i forgot to announce the workflow change